### DISCUSSION TITLE
How to use facet package with pretrained sklearn pipeline ?

### DISCUSSION THREAD

**Discussion by Paroag** [BCG-Gamma/facet#366] [Q&A] [answered]
I'm working on outlier detection, and using sklearn pipelines in my project. I currently compute shap values for my outliers to better understand how my underlying models use my features.

I would love to dig further into explainability by exploring the synergy and redundancy as defined in facet, however I noticed in most examples that the facet package uses sklearndf as the framework for ML models.

Is there a way to use this package with standard sklearn pipeline ? Any recommandation on how to use this package is very appreciated as i'm new to it.

**Comment by j-ittner** [comment, marked as answer]
Hi @Paroag , not yet but we're considering enhancing the `LearnerInspector` to also accept native scikit-learn pipelines. If you want, create an issue & we'll keep you up to date on progress! Thanks!

**Reply by Paroag** [reply]
Thanks for the quick answer !

**Reply by j-ittner** [reply]
Update: this will be a feature in FACET 2.1, see PR #369
